spec/factories/users.rb

```diff
@@ -43,7 +43,7 @@
 
     factory :single_academy_trust_user do
       association :responsible_body, factory: %i[trust single_academy_trust]
-      school { build(:school, :academy, responsible_body: responsible_body) }
+      schools { build_list(:school, 1, :academy, responsible_body: responsible_body) }
```

Comment:
Is it worth using `build_list` to build a list of one element?

Reply:
yes, because using build alone the rspec fails